test/integration/teamOps.test.js

```diff
@@ -21,7 +21,7 @@ lab.experiment('TeamOps', () => {
   let users
   let policies = []
   lab.before(done => {
-    udaru.users.list({organizationId: 'WONKA'}, (err, fetchedUsers) => {
+    udaru.users.list({ organizationId: 'WONKA' }, (err, fetchedUsers) => {
```

Comment:
notice the extra space enforced throughout, should this be enforced across the board in our editors... mine allows for {x: y} and { x: y } for example once the spacing is consistent on either side?